app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_to_arr = command.split()
+    if len(command_to_arr) != 3 or command_to_arr[0] != "cp":
+        return
+
+    source = command_to_arr[1]
+    destination = command_to_arr[2]
+
+    if source == destination:
+        return
```

Comment:
Prefer unpacking in retrieving arguments over using indexing 
```suggestion
    command_name, source, destination, *_ = command.split()
    if command_name != "cp" or source == destination:
        return
```